pandas/core/indexes/base.py

```diff
@@ -1339,6 +1342,237 @@ def rename(self, name, inplace=False):
         """
         return self.set_names([name], inplace=inplace)
 
+    def replace(
+        self,
+        to_replace=None,
+        value=None,
+        inplace=False,
+        limit=None,
+        regex=False,
+        method="pad",
+    ):
```

Comment:
Can you add a docstring

Reply:
https://github.com/pandas-dev/pandas/pull/32542#issuecomment-646993103